frontend/src/components/Common/MapModal.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Help.css';
+
+/**
+ *
+ * @param {*} props
+ * @return {*}
+ */
+function MapModal(props) {
+  return props.trigger ? (
+    <div className='Help'>
+      <div className='Help-inner'>
+        <iframe
+          src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3192.588180029826!2d174.7669186152492!3d-36.85233777993783!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x6d0d47e383f32253%3A0xbd49f61f758a9e5b!2sThe%20University%20of%20Auckland!5e0!3m2!1sen!2snz!4v1615862553109!5m2!1sen!2snz"
```

Comment:
I'd recommend passing this in as props rather than hardcoding this in (even though I know it's temporary so far)